core/src/test/scala/org/apache/spark/serializer/KryoSerializerSuite.scala

```diff
@@ -32,6 +32,36 @@ class KryoSerializerSuite extends FunSuite with SharedSparkContext {
   conf.set("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
   conf.set("spark.kryo.registrator", classOf[MyRegistrator].getName)
 
+  test("configuration limits") {
```

Comment:
General style note: this test has a _lot_ of repetition in it; the fact that we have four conf variables, conf1, conf2, conf3, and conf4, suggests that this could have benefitted from an inline `def` that abstracts away the actual configuration manipulation and KryoSerializer instantiation.  Either this or whitespace between the individual cases would make this test easier to quickly scan through.